test/member_test.rb

```diff
@@ -13,4 +13,12 @@
   it 'should have a name field with the name of the member' do
     subject.name.must_equal 'George Vella'
   end
+
+  it 'should have an electoral history field' do
+    subject.must_respond_to :electoral_history
+  end
+
+  it 'should return a hash from electoral_history' do
+    subject.electoral_history.must_be_kind_of Hash
```

Comment:
I'm not totally convinced that both of these tests are useful/valuable — the first is covered entirely by the second — but I guess there's no real harm in it.